src/Components/HomePage/Homepage.js

```diff
@@ -97,16 +97,16 @@ export const Homepage = (props) => {
       <ActivityCards />
 
       <section className="team-section">
-        <h1 className="team-title">A passionate team</h1>
+        <h1 className="team-title">What activities interest you?</h1>
```

Comment:
I think we can just get rid of this `section` because we are going to be rendering the `ActivityCards` component and it seems repetitive to have this in there.  Just an idea! Maybe less is more in this case.

Reply:
This section is just a content filler and I think it helps the viewer to have a little insight about our application without getting too much into details. I personally lean towards keeping it just because the user does not have to interact with the `find activity button` or ` activity links` on the nav bar to learn about the application. @BlakeDonn what do you think?